src/panels.tsx

```diff
@@ -321,6 +321,16 @@ class PSMain extends preact.Component {
 	}
 	handleButtonClick(elem: HTMLButtonElement) {
 		switch (elem.name) {
+		case 'finduser':
+			PS.addRoom({
+				id: 'finduser' as RoomID,
+				type: 'prompt',
+				message: 'Find a user',
+				parentElem: elem,
+				buttonName: "Open",
+			});
+			PS.update();
+			return true;
```

Comment:
Seems like this should be a link to the `finduser` room, rather than needing to manually dispatch it?

Reply:
Like just use `joinRoom` rather than `finduser`.

Reply:
OK.

Reply:
On second consideration, would that work? It would remove access to being able to set the message / buttonName params, no?